server/lib/status-check.js

```diff
@@ -26,23 +25,6 @@ export default function (req: Request, res: Response) {
 
   if (!agent.isQueryStringConfigured()) {
     pushMessage("Query is not configured");
-  } else if (req.agent.areConnectionParametersConfigured()) {
-    agent = new SyncAgent(req.hull);
-    promises.push(agent.runQuery(agent.getQuery(), { limit: 1, timeout: 3000 }).then(result => {
-      if (result.entries && result.entries.length === 0) {
-        let changeStatusTo = "warning";
-        if (status === "error") {
-          changeStatusTo = "error";
-        }
-        pushMessage("Database does not return any rows for saved query", changeStatusTo);
-      }
-
-      if (result.errors) {
-        pushMessage(`Results have invalid format: ${result.errors.join("\n")}`);
```

Comment:
Are we reporting those errors now on the `/sync` run  ?

Reply:
Tried to do so, but it's not that easy. The runQuery result is enriched with fields information too. So passing just one record instead is not a solution.